assets/css/pages.css

```
@@ -9995,26 +9995,26 @@ h3:first-child {
   display: grid;
   grid-template-columns: 100%;
 }
-.c1pbo4kt[class][class] {
+.card__slides_item_wrapper[class][class] {
```

```suggestion
.card__slides-item-wrapper[class][class] {
```
use hypen. papalitan nalang din po sa iba